### PR TITLE
ldid: submission

### PR DESCRIPTION
**Pull request by i0ntempest**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.4 19E287
Xcode 11.4.1 11E503a 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/ldid/Portfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description         Link Identity Editor
+long_description    Link Identity Editor lets you manipulate the signature block in a Mach-O binary.
+
+homepage            https://cydia.saurik.com/info/ldid/
+
+depends_lib-append  port:libxml2 \
+                    path:lib/libssl.dylib:openssl
+
+fetch.type          git
+git.url             --recursive --depth 2000 https://git.saurik.com/ldid.git
```
Comment (chrstphrchvz) [inline devel/ldid/Portfile:23]: Why is `--depth 2000` necessary?

I'm not sure `git.url` should be used to pass extra options to `git clone`, even if it works; this is not something any other ports appear to do. A few ports either define their own fetch or post-fetch phases if they need more control over `git clone`. But maybe this indicates a lack of `git.*` options to do so.

**Comment by i0ntempest** [comment]
I copied the command from another portfile so I thought it’s needed. I’ll change it later to use the documented way for dealing with submodules (post-fetch).

File: devel/ldid/Portfile (hunk 1 above)
Comment (i0ntempest) [reply devel/ldid/Portfile:23]: I copied the command from another portfile so I thought it’s needed. I’ll change it later to use the documented way for dealing with submodules (post-fetch).

File: devel/ldid/Portfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           xcode 1.0
+
+name                ldid
+version             2.1.2
+set git_commit      c2f8abf013b22c335f44241a6a552a7767e73419
```
Comment (mf2k) [inline devel/ldid/Portfile:8]: Use the existing `git.branch` variable instead of defining your own.

Comment (mf2k) [reply devel/ldid/Portfile:8]: Never mind. That makes sense.

**Comment by i0ntempest** [comment]
@chrstphrchvz Review please? For some reason re-request button doesn't do anything.

**Comment by chrstphrchvz** [comment]
> @chrstphrchvz Review please?

Looks OK to me, or I at least haven't thought of any other suggestions. More experienced folks might offer further advice and/or scrutiny.

**Pull request merged**